packages/sync-repo-settings/src/teams.json

```diff
@@ -0,0 +1,8 @@
+{
+    "java": [
+        {
+            "slug": "java-samples-reviewers",
```

Comment:
Per our discussion in the team meeting today, I love the idea of this being generalizable across languages.

Reply:
For now, I think only java-samples-reviewers exists and we would like this group on all the java repos. We can refactor this one the internal design is approved.